openni/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# OpenCV
+find_package(OpenCV REQUIRED)
```

Comment:
in order to be consistent with other plugins, you might want to specify the OpenCV version